### ISSUE TITLE
Cannot open a database that uses a keyfile

### ISSUE THREAD

**Issue by hThoreau** [keepassxreboot/keepassxc#2775]
## Expected Behavior

List contents of the database root folder

## Current Behavior

```
> keepassxc-cli ls --key-file .passwords/passwords.k{ey,dbx}
Insert password to unlock .passwords/passwords.kdbx: 
Error while parsing the database: Wrong key or database file is corrupt.
>      
```
When using the GUI though, the database is opened.

## Steps to Reproduce
[NOTE]: # ( Provide a link to a live example, or an unambiguous set of steps to )
[NOTE]: # ( reproduce this bug. Include code to reproduce, if relevant )
1. Create a database using the GUI interface (`keepassxc`) with a password and keyfile
2. Add a subfolder and one entry in it
3. `keepassxc-cli ls --key-file .passwords/passwords.k{ey,dbx}`

## Debug Info

KeePassXC - keepassxc 2.3.4
Operating system: Parrot GNU/Linux 4.5
CPU architecture: x86_64
Kernel: 4.19.0-parrot1-13t-amd64

Enabled extensions:
None

**Comment by hThoreau** [comment]
I just recreated the database using [passhole](https://github.com/Evidlo/passhole), and it now works... Maybe it changed its filetype? is `Keepass password database 2.x KDBX` the default format for KeepassXC?

**Comment by droidmonkey** [comment]
I'll try to replicate, but it is a little hard to follow your sequence. Please provide more details on the creation process.
